administrator/language/de-DE/com_content.ini

```
@@ -65,6 +65,7 @@ COM_CONTENT_FIELD_IMAGE_ALT_EMPTY_DESC="Dekoratives Bild - keine Beschreibung er
 COM_CONTENT_FIELD_IMAGE_ALT_EMPTY_LABEL="Keine Beschreibung"
 COM_CONTENT_FIELD_IMAGE_ALT_LABEL="Bildbeschreibung (Alternativer Text)"
 COM_CONTENT_FIELD_IMAGE_CAPTION_LABEL="Bildunterschrift"
+COM_CONTENT_FIELD_IMAGE_CLASS_DESC="Es kann eine beliebige CSS-Klasse für eigene Styling-Ideen hinzugefügt werden.<br>Für die Bildposition beispielsweise float-start und float-end verwenden."
```

```suggestion
COM_CONTENT_FIELD_IMAGE_CLASS_DESC="Es kann eine beliebige CSS-Klasse für eigene Styling-Ideen hinzugefügt werden.<br>Für die Bildposition kann z.B. 'float-start' und 'float-end' verwendet werden."
```
Ggfs etwas flüssiger?